### DISCUSSION TITLE
Audible and audiobookshelf integraion

### DISCUSSION THREAD

**Discussion by decoylad** [IgnisDa/ryot#388] [Q&A] [answered]
I know these are integrated, but I'm a little confused,  it's being hosted in another docker container on the same nas.  For the base url do I use the ip without the port ie http://10.0.0.1  or with the port?  http://10.0.0.1:99 ?   I do have it accessible externally via synology RP at something like https://audio.stuff.com:port   Token I've got.   

Additionally I saw the twitch/igdb config for the video game section, are there similar environment variables that need to be set for audiobooks?

Thanks.

**Comment by IgnisDa** [comment, marked as answer]
1. I am not very familiar with docker internal networking. I would suggest trying all of them one by one and then using the second button in the miscellaneous settings page to force a yank from ABS to see if it works.
2. Nope. Audiobooks data is taken from Audible which has an open api.

**Reply by IgnisDa** [reply]
Regarding 2, there are env variables to change the locale used for Audible but you probably don't need that.
